lib/tasks/verify_historic_duplicates.rake

```diff
@@ -0,0 +1,12 @@
+task verify_historic_duplicates: :environment do
+  Statement.original.each do |statement|
+    next unless statement.duplicate_statements.any?
+    latest_verification = statement.latest_verification
+    statement.duplicate_statements.each do |duplicate_statement|
+      duplicate_statement.verifications.find_or_create_by!(
+        user: latest_verification.user,
```

Comment:
If the statement hasn't been verified then `latest_verification` will be `nil` and this line will blow up, should probably check its non-nil outside the loop.